transmogrifier/sources/ead.py

```diff
@@ -117,6 +119,80 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
 
         # format field not used in EAD
 
+        # holdings
```

Comment:
Was the `funding_information` field skipped intentionally? If so, it at least needs a placeholder comment.

Reply:
That was an oversight, I will add it on the next PR. Good catch

Reply:
I had previously mapped the `sponsor` element to `funding_information` but after another look and given our previous conversations about data outside of `archdesc` that relates only to the finding aid itself, I don't think `funding_information` should be mapped for EAD but I am flexible if you disagree. The values mostly refer to funding for the processing of collections which doesn't seem to fit the usage of this field in other sources.